src/test/java/org/jabref/logic/search/query/SearchQueryExtractorConversionTest.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.logic.search.query;
+
+import java.util.Set;
+import java.util.stream.Stream;
+
+import org.jabref.model.search.query.SearchQuery;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SearchQueryExtractorConversionTest {
+    public static Stream<Arguments> testSearchConversion() {
+        return Stream.of(
+                Arguments.of(Set.of("term"), "term"),
+                Arguments.of(Set.of("regex.*term"), "regex.*term"),
+                Arguments.of(Set.of("term"), "any = term"),
```

Comment:
According to https://docs.jabref.org/finding-sorting-and-cleaning-entries/search#pseudo-fields, it is `anyfield`, not just `any`. I think, we shoud keep `anyfield`. At least as synonym.

Reply:
Both of them are working see https://github.com/JabRef/jabref/blob/c7baf048e4056a292e37e720dd1a481810a2a6c2/src/main/java/org/jabref/logic/search/query/SearchToSqlVisitor.java#L201